ans2ask-api/routes/googleScraping.js

```diff
@@ -0,0 +1,50 @@
+import unirest from "unirest";
+import cheerio from "cheerio";
+import express from 'express';
+
+const router = express.Router();
+
+const getDataFromGoogleSearchResults = (query) => {
+    return unirest
+        .get(`https://www.google.com/search?q=${query}+youtube+courses&tbm=vid`)
+        .headers({
+            "User-Agent":
+                "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Safari/537.36, Custom-Agent/1.0"
+        })
+        .then((response) => {
+            let $ = cheerio.load(response.body);
+
+            let titles = [];
+            let links = [];
+
+            $(".DhN8Cf > a > h3").each((i, el) => {
+                titles[i] = $(el).text();
+            });
+            $(".DhN8Cf > a").each((i, el) => {
+                links[i] = $(el).attr("href");
+            });
```

Comment:
May I ask what this used for? Filtering non-title text?
If so can we make a subFunc with proper naming

`getTitleFromResponse(){
  $(".DhN8Cf > a > h3").each((i, el) => {
                titles[i] = $(el).text();
            });
}`